src/pages/ProfilePage.js

```diff
@@ -121,7 +121,7 @@ function ProfilePage(props) {
     const phoneNumber = getPhoneNumber(details);
     const phoneOrEmail = isSMSLogin ? getPhoneNumber(details) : login;
 
-    const isCurrentUser = _.keys(props.loginList).includes(login);
```

Comment:
If loginList is not used anywhere remove it from onyx subscription

Reply:
Thanks. Done.